client/client.go

```diff
@@ -844,10 +848,37 @@ func (c *client) GetMinTS(ctx context.Context) (physical int64, logical int64, e
 		// returning the default timeline should be fine.
 		return c.GetTS(ctx)
 	case pdpb.ServiceMode_API_SVC_MODE:
-		return tsoClient.getMinTS(ctx)
 	default:
 		return 0, 0, errs.ErrClientGetMinTSO.FastGenByArgs("undefined service mode")
 	}
+
+	// Call GetMinTS API to get the minimal TS from the API leader.
+	protoClient := c.getClient()
+	if protoClient == nil {
+		return 0, 0, errs.ErrClientGetProtoClient
+	}
+
+	resp, err := protoClient.GetMinTS(ctx, &pdpb.GetMinTSRequest{
+		Header: c.requestHeader(),
+	})
+	if err != nil {
+		if strings.Contains(err.Error(), "Unimplemented") {
```

Comment:
Do we need to add a test for this error?

Reply:
It's pretty straightforward, not necessary?